types/multy/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for multy 0.1
+// Project: https://github.com/eduardorfs/multy
+// Definitions by: Jan Dolezel <https://github.com/dolezel>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import * as Koa from "koa";
+import * as busboy from "busboy";
+
+declare module "koa" {
+    interface Request {
+        body: any;
+    }
+}
+
+declare function multy(opts?: busboy.BusboyConfig): Koa.Middleware;
+
+declare namespace multy {}
```

Comment:
This is an inappropriate hack. [Please read more here from the DefinitelyTyped README](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/fbba9f55beab742b9d1c0fa9ef714e0771ad9d05#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports)

Reply:
I added alias for options, so user can use type directly from this package.